python/lsst/ts/dcs/dcs_csc.py

```diff
@@ -0,0 +1,55 @@
+# import asyncio
+import pathlib
+from lsst.ts import salobj
+
+__all__ = ['DCSCSC']
+
+
+class DCSCSC(salobj.ConfigurableCsc):
+    """
+    Upper level Commandable SAL Component to interface with the LSST Dome lower level components.
+    """
+
+    def __init__(
+            self,
+            config_dir=None,
+            initial_state=salobj.State.STANDBY,
+            simulation_mode=0,
+    ):
+        """
+        Initialize DIMM CSC.
+        """
```

Comment:
We basically use numpydoc, which (surprisingly to me) documents the constructor in the main class doc string, and the `__init__` method has no doc string.

Reply:
(Is this actually the DIMM CSC? I assumed it was the new dome controller.)

Reply:
Thanks for the comments Russel. I renamed the CSC to DomeCsc. The reference to DIMM is a copy and paste error since I am using the DIMM CSC as an example as well, which has a doc string inside the __init__ method. I removed the comment all together.